lib/app/views/app_page.dart

```diff
@@ -684,8 +684,7 @@ class _AppPageState extends ConsumerState<AppPage> {
                 ),
           actions: [
             if (widget.actionButtonBuilder == null &&
-                (widget.keyActionsBuilder != null &&
-                    (!hasManage || !showDetailView)))
+                (widget.keyActionsBuilder != null && (!hasManage)))
```

Comment:
Maybe remove the nested parentheses around `!hasManage`  to improve readability?

Reply:
Agreed, fix coming up!